build.gradle

```diff
@@ -268,7 +268,7 @@ subprojects {
 // the dependency is added.
 gradle.projectsEvaluated {
   allprojects {
-    if (project.path == ':test:framework') {
+    if (project.path == ':test:framework' || project.path == ':client:test') {
       // :test:framework:test cannot run before and after :core:test
```

Comment:
Sorry, `:client:test` cannot run before `:core:test`. Basically, if we are going to leave a comment explaining why `:test:framework` needs this exemption we should probably explain the `:client:test` too.